Orange/statistics/util.py

```diff
@@ -44,6 +44,22 @@ def sparse_has_zeros(x):
     return np.prod(x.shape) != x.nnz
 
 
+def sparse_zero_weights(x, weights):
+    """ Extract the weight values of all zeros in a sparse matrix. """
+    if not sp.issparse(x):
+        raise TypeError('The matrix provided was not sparse.')
+
+    if weights.ndim == 1:
+        n_items = np.prod(x.shape)
+        zero_indices = np.setdiff1d(np.arange(n_items), x.indices, assume_unique=True)
```

Comment:
Does `x.indices` works here for both `csc` and `csr`? If not, please cast it to whatever (csc/csr) is required beforhand.